index.html

```diff
@@ -0,0 +1,182 @@
+                    <a class="navbar-brand logo" href="#"><img class="twitter" src="src/img/twitter.svg" alt=""></a>
+                </div>
+
+                <div class="collapse navbar-collapse" id="navbarSupportContent">
+                    <ul class="navbar-nav me-auto mb-2 mb-lg-0 justify-content-end">
+                        <li class="nav-item">
+                            <a class="nav-link text-white" href="#">English</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link text-white" href="#">My page</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link text-white" href="#">Logout</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+
+        <nav class="navbar navbar-expand-lg navbar-light container">
+            <div class="container-fluid">
+                <a class="navbar-brand logo" href="index.html"><i class="fas fa-tram fa-3x">travel</i></a>
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse"
+                    data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent"
+                    aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                    <ul class="navbar-nav me-auto mb-2 mb-lg-0 justify-content-end">
+                        <li class="nav-item">
+                            <a class="nav-link" aria-current="page" href="about.html">About</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Program</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Join</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Sponsor</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">News</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="btn btn-outline-danger border-4">CC Campaign</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="bg-image">
+            <div class="container pt-5">
+                <h1 class="title">"Hello! Sharing world"</h1>
+                <h2 class="sub-title">CREATIVE COMMONS GLOBAL SUMMIT 2022</h2>
```

Comment:
The pages should look almost identical to the original design. Please make sure to add text effect for the h2 as with the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).